traffic_ops_ort/traffic_ops_ort.pl

```diff
@@ -48,6 +33,8 @@
 my $dns_local_bind = 0;
 my $disable_parent_config_comments = 0;
 
+print "ERROR traffic_ops_ort.pl is deprecated and will be removed in the next major version! Please upgrade to t3c\n";
```

Comment:
Should this be going to stderr?

Reply:
It's following the pattern of the existing ort.pl, e.g. https://github.com/apache/trafficcontrol/blob/master/traffic_ops_ort/traffic_ops_ort.pl#L388

I think it should follow the previous pattern as closely as possible, so as to be picked up automatically by any existing tooling users might have.

Reply:
That sounds fine to me